sweep/sweeper.go

```diff
@@ -398,10 +402,12 @@ func (s *UtxoSweeper) SweepInput(input input.Input,
 	}
 
 	log.Infof("Sweep request received: out_point=%v, witness_type=%v, "+
-		"time_lock=%v, amount=%v, fee_preference=%v", input.OutPoint(),
+		"time_lock=%v, amount=%v, fee_preference=%v, "+
+		"force=%v",
+		input.OutPoint(),
```

Comment:
Nit: looks like we can save on some whitespace by merging these lines with the above.

Reply:
fixed